libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -433,6 +433,24 @@ void ContractLevelChecker::checkHashCollisions(ContractDefinition const& _contra
 			);
 		hashes.insert(hash);
 	}
+
+	map<uint32_t, SourceLocation> errorHashes;
+	for (ErrorDefinition const* error: _contract.interfaceErrors())
+	{
+		if (!error->functionType(true)->interfaceFunctionType())
+			// Will create an error later on.
```

Comment:
```suggestion
			// Not having an external type will result in a compilation error later anyway so we can ignore it here.
```

At first I interpreted it as a TODO. I'd reword it to make it clearer: